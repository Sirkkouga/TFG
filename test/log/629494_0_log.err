INFO - 06/04/24 15:51:03 - 0:00:00 - Extracting data from /home/usuaris/veu/javier.abella/TFG/test/java.000.json.gz
INFO - 06/04/24 15:51:03 - 0:00:00 - Pre-filtered 0 json lines among 50 (0.00%)
INFO - 06/04/24 15:51:03 - 0:00:00 - Number of lines to process: 50
INFO - 06/04/24 15:51:04 - 0:00:01 - Time elapsed: 0.61
WARNING - 06/04/24 15:51:04 - 0:00:01 - Tokenization of /home/usuaris/veu/javier.abella/TFG/test/java.000.json.gz:0 errors out of 50 lines(0.00%)
WARNING - 06/04/24 15:51:04 - 0:00:01 - Tokenization of /home/usuaris/veu/javier.abella/TFG/test/java.000.json.gz:0 timeouts out of 50 lines(0.00%)
WARNING - 06/04/24 15:51:04 - 0:00:01 - Tokenization of /home/usuaris/veu/javier.abella/TFG/test/java.000.json.gz:0 filtered examples in 50 lines(0.00%)
WARNING - 06/04/24 15:51:04 - 0:00:01 - Tokenization of /home/usuaris/veu/javier.abella/TFG/test/java.000.json.gz:0 multiline codes 50 lines(0.00%)
submitit WARNING (2024-06-04 15:51:04,515) - Bypassing signal SIGTERM
WARNING - 06/04/24 15:51:04 - 0:00:01 - Bypassing signal SIGTERM
INFO - 06/04/24 15:51:04 - 0:00:01 - Job completed successfully
INFO - 06/04/24 15:51:04 - 0:00:01 - Exiting after successful completion
