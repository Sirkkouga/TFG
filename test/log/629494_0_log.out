submitit INFO (2024-06-04 15:51:02,081) - Starting with JobEnvironment(job_id=629494, hostname=veuc09, local_rank=0(1), node=0(1), global_rank=0(1))
submitit INFO (2024-06-04 15:51:02,081) - Loading pickle: /home/usuaris/veu/javier.abella/TFG/test/log/629494_submitted.pkl
adding to path /home/usuaris/veu/javier.abella/TFG
Using /home/usuaris/veu/javier.abella/data/CodeGen/code_evaluation/tmp_tests_folder for temporary files.
Registered languages: {'treesitterlang': <class 'codegen_sources.preprocessing.lang_processors.tree_sitter_processor.TreeSitterLangProcessor'>, 'java': <class 'codegen_sources.preprocessing.lang_processors.java_processor.JavaProcessor'>, 'cpp': <class 'codegen_sources.preprocessing.lang_processors.cpp_processor.CppProcessor'>, 'go': <class 'codegen_sources.preprocessing.lang_processors.go_processor.GoProcessor'>, 'python': <class 'codegen_sources.preprocessing.lang_processors.python_processor.PythonProcessor'>, 'kotlin': <class 'codegen_sources.preprocessing.lang_processors.kotlin_processor.KotlinProcessor'>, 'bytecode': <class 'codegen_sources.preprocessing.lang_processors.bytecode_processor.BytecodeProcessor'>}
submitit WARNING (2024-06-04 15:51:04,515) - Bypassing signal SIGTERM
submitit INFO (2024-06-04 15:51:04,559) - Job completed successfully
submitit INFO (2024-06-04 15:51:04,560) - Exiting after successful completion
